pkg/operation/botanist/component/etcd/bootstrap.go

```diff
@@ -319,7 +319,7 @@ func (b *bootstrapper) Deploy(ctx context.Context) error {
 }
 
 func getDruidDeployCommands(gardenletConf *config.GardenletConfiguration) []string {
-	command := []string{"" + "/bin/etcd-druid"}
+	command := []string{"" + "/etcd-druid"}
```

Comment:
I'm surprised that there is no breaking release note in etcd-druid about this change.

Reply:
This was the PR that made the change. I too am surprised that the release note is missing. I'll add it now. Thanks for the catch.